package.json

```
@@ -0,0 +1,65 @@
+{
+  "name": "docusaurus-website",
+  "version": "0.0.0",
```

Let's set at least `0.1.0` 😄 We don't need to change it later.
```suggestion
  "version": "0.1.0",
```